src/searchindex_js.cpp

```diff
@@ -284,42 +283,41 @@ static void addMemberToSearchIndex(const MemberDef *md)
       )
      )
   {
-    QCString n = md->name();
-    if (!n.isEmpty())
+    if (!md->name().isEmpty())
```

Comment:
I don't think this is necessarily an improvement. `n` was computed only once before and then reused, and now `md->name()` is called multiple times, which leads to code duplication, and is probably less efficient as well. Same for the case above.